FreeAPS/Sources/Modules/Settings/View/SettingsRootView.swift

```diff
@@ -61,6 +61,12 @@ extension Settings {
                                     .frame(maxWidth: .infinity, alignment: .trailing)
                                     .buttonStyle(.borderedProminent)
                             }
+                            HStack {
+                                Text("Delete Pump State nlist")
```

Comment:
Let’s remove the debug option for deleting plist before merging, to reduce risk of accidental removals. 

Question: is nlist synonymous with plist?

Reply:
Commented out with a comment pointing to this PR :)